Podcast/RecommendedTagsTableViewCell.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  RecommendedTagsTableViewCell.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 2/19/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+protocol RecommendedTagsTableViewCellDataSource {
+    func recommendedTagsTableViewCell(dataForItemAt indexPath: IndexPath) -> String
```

Comment:
same issue, shouldn't we tell whoever implements the protocol which tag the user tapped?